purchase_stock_picking_return_invoicing/README.rst

```diff
@@ -20,76 +20,66 @@ In the purchase order you are able to display, for each line:
 * Received Quantity and Returned Quantity, as separate fields.
 
 
-
 Usage
 =====
 
 Case 1: When you return to a supplier some products, and you have not yet
 received the bill from the supplier
 
 #. Go to *Purchases > Purchase > Purchase Orders > Create*.
-#. Choose a supplier and add a product whose *Invoicing Policy* is *On Received
-   quantities*, and input some quantity to purchase.
+#. Choose a supplier and add a product whose *Invoicing Policy* is
+   *Delivered quantities*, and input some quantity to purchase.
```

Comment:
What was wrong in the orignal sentence was 'Invoicing Policy', actually the field you have to modify is 'Control Purchase Bills' and set it to 'On received quantities'.